Essentials/src/messages.properties

```diff
@@ -20,10 +20,10 @@ antiBuildPlace=\u00a74You are not permitted to place\u00a7c {0} \u00a74here.
 antiBuildUse=\u00a74You are not permitted to use\u00a7c {0}\u00a74.
 autoAfkKickReason=You have been kicked for idling more than {0} minutes.
 backAfterDeath=\u00a76Use the /back command to return to your death point.
+backUsageMsg=\u00a76Returning to previous location.
```

Comment:
You guys are changing a lot of messages here that don't seem relevant to the PR. Can you guys try to minimize the diff in the messages files?